### PR TITLE
seccomp: add LoongArch 64bit support

### PR DESCRIPTION
**Pull request by yetist**
The test results on ArchLinux(LoongArch) are as follows:

```
Summary of Failures:

 802/1201 systemd:journal / test-journal-verify                            TIMEOUT         90.01s   killed by signal 15 SIGTERM

Ok:                 1179
Expected Fail:      0   
Fail:               0   
Unexpected Pass:    0   
Skipped:            21  
Timeout:            1   
```

### CODE REVIEW

File: src/shared/seccomp-util.c [hunk 1]
```diff
@@ -47,6 +47,8 @@ uint32_t seccomp_local_archs[] = {
                 SCMP_ARCH_AARCH64,     /* native */
 #elif defined(__arm__)
                 SCMP_ARCH_ARM,
+#elif defined(__loongarch_lp64)
```
Comment (yuwata) [inline src/shared/seccomp-util.c:50]: Why not `__loongarch64` ?

Comment (yetist) [reply src/shared/seccomp-util.c:50]: Please read the end of this document, https://loongson.github.io/LoongArch-Documentation/LoongArch-toolchain-conventions-EN.html#_cc_preprocessor_built_in_macro_definitions

Comment (bluca) [reply src/shared/seccomp-util.c:50]: `if defined(__loongarch64)` is used elsewhere, do those need to be updated?

Comment (yetist) [reply src/shared/seccomp-util.c:50]: It is best if __loongarch64 can be updated, and if it cannot, it will still be available at least for a while

Comment (bluca) [reply src/shared/seccomp-util.c:50]: could you please add a commit to do so then?

Comment (yetist) [reply src/shared/seccomp-util.c:50]: Done.

**Review by bluca — APPROVED** [review]

**Pull request merged**